bin/build-cored-release

```diff
@@ -48,7 +48,7 @@ DATE=${DATE:-`date +%s`} # date can be set via envvar
 ldflags="-X main.buildTag=$releaseRef -X main.buildCommit=$commit -X main.buildDate=$DATE"
 
 go build\
-  -tags 'plain_http'\
+  -tags 'plain_http loopback_auth'\
```

Comment:
Nah, I think it's good as-is. We are prob going to be reworking the release tools soon anyway.